app/models/pregnancy.rb

```diff
@@ -55,6 +61,8 @@ class Pregnancy
             presence: true
   validates_associated :patient, on: :create
 
```

Comment:
Go ahead and remove lines 64-72 here since we're replacing them with `Auditable`.